### PR TITLE
Update npm packages

### PR DESCRIPTION
**Pull request by maxno-kivra**
I just ran `npm update`, which respects package.json version constraints.

I was thinking of upgrading the the very latest, to keep up with things. I'm guessing it would still be safe considering how small this extension is. What do you think?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   },
   "main": "./client/out/client.js",
   "scripts": {
-    "package": "vsce package --yarn",
+    "package": "vsce package",
```
Comment (maxno-kivra) [inline package.json:21]: The dev readme doesn't use yarn, so I fixed this script. Should maybe be in a separate commit, but it's a pretty small fix.

**Comment by robertoaloi** [comment]
Thanks!
